app/upgrades.go

```diff
@@ -0,0 +1,32 @@
+package app
+
+import (
+	"fmt"
+
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	icacontrollertypes "github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/controller/types"
+)
+
+func (app *App) RegisterUpgradeHandlers() {
+	planName := "v0.9.0"
+	app.UpgradeKeeper.SetUpgradeHandler(planName, func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
```

Comment:
ibc-go upgrade notes aren't there yet, so not sure whether there's something else to be set.
for SDK v0.46, there some new params, but probably no relevant to cronos